lib/std/builtin.zig

```diff
@@ -183,6 +183,14 @@ pub const SourceLocation = struct {
     column: u32,
 };
 
+/// This data structure is used by the Zig language code generation and
+/// therefore must be kept in sync with the compiler implementation.
+pub const DeclarationInfo = struct {
```

Comment:
```suggestion
pub const Declaration = struct {
```
To match `Type` below.

Reply:
That would conflict with the the already existing Declaration type inside of Type, but maybe Type.Declaration could be renamed to something else but I'm unsure to what that would be.

Reply:
No need to rename, just use `Type.Declaration` instead of plain `Declaration`.